addons/sale/static/tests/sale_product_field.test.js

```diff
@@ -86,7 +86,7 @@ test("Hide product name if its not translated", async () => {
         partner_id: serverState.partnerId,
         order_line: [Command.create({
             product_id: product.id,
-            name: product.name.concat("\nA description"),
+            name: "A description",
```

Comment:
Why this change?
What is effectively stored in the name is the concatenation of the name and the description, so I don't understand why you'd need to change this :thinking: 

cc: @lipiraux